src/components/ItemLayout/ItemLayout.tsx

```diff
@@ -175,8 +187,8 @@ class ItemLayout extends UIComponent<ReactProps<ItemLayoutProps>, any> {
     const mainArea = renderMainArea(this.props, this.state, classes)
     const endArea = endMedia
 
-    const mergedMediaClasses = cx('ui-item-layout__media', classes.media)
```

Comment:
Ups, breaking changes introduced. It seems that the classNames of this slots were not crated correctly...